docs/_templates/page.html

```diff
@@ -13,6 +13,12 @@
                 $(".sticky-top .bd-toc").css("visibility", "hidden");
             }
         });
+        $(document).keydown(function(event) {
+            if (event.ctrlKey && event.which == 75) {
```

Comment:
- Consider macOS, not just Windows and Linux.
- Let's use readable names of keys, instead of obtuse ASCII codes.
- [`which` is deprecated](https://developer.mozilla.org/en-US/docs/Web/API/UIEvent/which).

```suggestion
            if ((event.ctrlKey || event.metaKey) && event.key == "k") {
```

The MDN docs are awesome for testing and referring to this stuff. Random sources are not your friend.

https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key#result

Reply:
We need to give users some visual indication that a keyboard shortcut even exists. There are two possibilities I can think of, but maybe @ksuess has better ideas.

1. Modify the `placeholder` attribute on both the sidebar Search box and Search page.
2. Add a `title` attribute.
3. Something else?

Reply:
I also think that we need some indication, so `placeholder` might be a good idea, for now, ig. and for another PR I have suggested this: 
> I am considering adding the `^k` to the right side of the search input field OR changing the placeholder to `use "ctrl+k" to set focus`. Before I implement this feature, I would like to get feedback from others. Please let me know your thoughts.

![Screenshot_20230102_125451](https://user-images.githubusercontent.com/99530996/210203679-88ff012f-9adb-4b12-8144-c2b206fedf4d.png)

Reply:
So far this is good.

However, we should strive to fully implement new features. Whether the display of the shortcut is part of this PR or another is up to you. I cannot merge this PR by itself because it needs to have the other part to be complete.

After thinking it through and following the inspiration of the [Crema docs](https://docs.cremawork.com/overview/folder-structure), here is my suggestion. The screenshot is on desktop.

![Screen Shot 2023-01-02 at 12 58 13 AM](https://user-images.githubusercontent.com/102112/210210903-a480a063-8290-491f-bd4c-d82b4582fd67.png)

- Insert a `span` with the HTML markup below for the sidebar search, but move the CSS from inline to our `custom.css`. I chose the colors based on the Plone palette and for sufficient contrast.
- Add a JavaScript that detects the platform and adapts the keyboard shortcut accordingly. For macOS and iOS, use `⌘K`, and for all others use `^K`.
- Apply this feature to both the sidebar search and Search page.
- On the Search page only, hide or remove the sidebar search input.
- When a user starts typing in the search input, hide the `span`. This will avoid a conflict with the clear icon of the input of `type="search"`.
- Adapt the CSS for both mobile and desktop through [media queries](https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries). The `font-size`, `padding`, and `position` attributes might need to be adjusted.
- On the search page, use `id="search-shortcut-page"`, to make it easy for CSS selectors.

```html
<span id="search-shortcut-sidebar" style="
    position: absolute;
    right: 20px;
    border-radius: 0.25em;
    background: #d9eef9;
    padding: 0.2em;
    color: #4a4a4a;
">⌘K</span>
```

Reply:
>  However, we should strive to fully implement new features. Whether the display of the shortcut is part of this PR or another is up to you. I cannot merge this PR by itself because it needs to have the other part to be complete.

Oh, No problem. Sure, then I'll make this PR complete by adding a display of the shortcut. I'll go through all these inputs you have given and I'll let you know when this PR is ready for review.
Thanks😊!

Reply:
> Insert a span with the HTML markup below for the sidebar search, but move the CSS from inline to our custom.css. I chose the colors based on the Plone palette and for sufficient contrast.

Hello, @stevepiercy I am sorry for the late response, but I'm trying to add a span element to [`search-field.html`](https://github.dev/pydata/pydata-sphinx-theme/blob/main/src/pydata_sphinx_theme/theme/pydata_sphinx_theme/components/search-field.html), but I'm having trouble finding it. I've looked in the usual places ( read [docs](https://sphinx-book-theme.readthedocs.io/en/stable/customize/sidebar-primary.html?highlight=search#default-sidebar-elements) already), but I'm unsure where else to look. Could you please help me locate it?

I can do that with JS but it may cause the UX problem IG.

Reply:
This is a Sphinx thing. A great overview of how Sphinx themes work is at https://www.sphinx-doc.org/en/master/templating.html.

You need to override the ancestral theme files, which are located in the Python virtual environment. Do not edit files in the virtual environment, but instead place a file with the same name and relative location as the ancestral themes in our local theme overrides in `docs/_templates`. For example, in `docs/_templates/layout.html`, `{% extends "sphinx_book_theme/layout.html" %}` means extend the parent theme file `layout.html` with the following stuff.

That should help you get started, but here are few more points.

`lib/python3.10/site-packages/sphinx_book_theme/theme/sphinx_book_theme/sections/sidebar.html` says `{%- for sidebartemplate in sidebars %}`.

In its config file `lib/python3.10/site-packages/sphinx_book_theme/theme/sphinx_book_theme/theme.conf`, an array of `sidebars` is defined as `sidebars = sidebar-logo.html, search-field.html, sbt-sidebar-nav.html`. If the file does not exist in the parent theme, go up a generation to the grandparent theme `lib/python3.10/site-packages/pydata_sphinx_theme/theme/pydata_sphinx_theme/search-field.html`.

The great-grandparent theme is `lib/python3.10/site-packages/sphinx/themes/basic` by virtue of `{%- extends "basic/layout.html" %}`.

As you can see, it takes some time to learn this stuff, but the good thing is that theming in Volto is very similar to that in Sphinx. It also often similar to other theming engines, so it is a good skill to acquire, as the experience is transferrable.

Reply:
> This is a Sphinx thing. A great overview of how Sphinx themes work is at https://www.sphinx-doc.org/en/master/templating.html.

Oh, I didn't look into templating section 😅.

I may require a bit of time (also since my semester break is over😖) to go through all this but I'll make sure to do it in a day or 2, because theming in Volto is also what I am interested in.

Thanks a lot for taking the time to help me with this issue. Your guidance is really helpful and I really appreciate it and its really means a lot to me (as a newbie).